src/main/java/ch/njol/skript/events/EvtItem.java

```diff
@@ -74,10 +75,19 @@ public class EvtItem extends SkriptEvent {
 						"\t\tset item of event-dropped item to a diamond")
 				.since("<i>unknown</i> (before 2.1), INSERT VERSION (entity)");
 		if (hasPrepareCraftEvent) { // Must be loaded before CraftItemEvent
-			Skript.registerEvent("Prepare Craft", EvtItem.class, PrepareItemCraftEvent.class, "[player] (preparing|beginning) craft[ing] [[of] %-itemtypes%]")
+			Class<? extends Event>[] events;
+			if (hasPlayerStonecutterRecipeSelectEvent)
+				events = CollectionUtils.array(PrepareItemCraftEvent.class, PlayerStonecutterRecipeSelectEvent.class);
+			else
+				events = CollectionUtils.array(PrepareItemCraftEvent.class);
```

Comment:
```suggestion
			if (hasPlayerStonecutterRecipeSelectEvent) {
				events = CollectionUtils.array(PrepareItemCraftEvent.class, PlayerStonecutterRecipeSelectEvent.class);
			} else {
				events = CollectionUtils.array(PrepareItemCraftEvent.class);
			}
```
Braces on chained if statements